frontend/src/employee-frontend/components/absences/Absences.tsx

```diff
@@ -114,34 +119,41 @@ export default React.memo(function Absences({
     }
   }, [absences, setTitle])
 
-  const updateCategories = useCallback(
-    (category: AbsenceCategory) => {
-      const included = selectedCategories.includes(category)
-      const without = selectedCategories.filter((item) => item !== category)
-      setSelectedCategories(included ? without : [category, ...without])
-    },
-    [selectedCategories, setSelectedCategories]
+  const showCategorySelection = useMemo(
+    () =>
+      selectedCells.map(({ parts }) => parts.length).some((count) => count > 1),
```

Comment:
This goes through the cells twice, unnecessarily:
```suggestion
      selectedCells.some(({ parts }) => parts.length > 1),
```